project-management/FujitsuMeetingDates

```diff
@@ -0,0 +1,23 @@
+# Fujitsu/Turing Sprint Meetings
+## Meetings Descriptions
+Each sprint is 9 days (approx. 2 work weeks long) therefore each of these meetings occurs every 2 weeks. 
+The whole Alan Turing and Fujitsu team are invited to these meetings and a member of the community will be included in retrospective meetings.
+*Backlog refinement* is carried out before each sprint to reprioritise the backlog of user stories (issues) ahead of the next sprint planning meeting.
+*Sprint Planning* is carried out at the start of each sprint and is when the user stories (issues) that are planned to be completed in that sprint are chosen.
+*Sprint retrospective* is carried out at the end of each sprint to consider how the sprint went and any changes that need to be made to the process before the next sprint.
+During this meeting a demo of the achievements of the previous sprint will be shown. 
```

Comment:
I don't think this is true.

Reply:
Yes, the demo will be available before the retrospective but it will be shown during the retrospective for anyone who hasn't had chance to view it/ to refresh peoples memories.